### ISSUE TITLE
Edit language on Chapter Ambassador dashboard [5,3,1.67]

### ISSUE THREAD

**Issue by rsgonzal** [Iridescent-CM/technovation-app#2612]
With the new language around Chapter Ambassadors, we want to make the dashboard make more sense.

![Screen Shot 2020-08-10 at 12 43 07 PM](https://user-images.githubusercontent.com/22801026/91239815-4065a280-e6f5-11ea-8614-9a71801b0aba.png)
- Promote your program to your region --> Promote your program to your area
- Add links to your regional program sites --> Add links to your local program sites
- Regional program introduction --> Local program introduction

_Bottom screen is after they've already filled out the above_
![Screen Shot 2020-08-25 at 5 12 13 PM](https://user-images.githubusercontent.com/22801026/91240183-237d9f00-e6f6-11ea-8003-3e203f6951da.png)
- Introduce yourself to your region --> Introduce yourself to your area

![Screen Shot 2020-08-25 at 5 10 40 PM](https://user-images.githubusercontent.com/22801026/91240114-edd8b600-e6f5-11ea-9168-0da21b57fdb8.png)
- View all participants in your region --> View all participants in your area

![Screen Shot 2020-08-25 at 5 11 31 PM](https://user-images.githubusercontent.com/22801026/91240156-0a74ee00-e6f6-11ea-9999-8eb77235e55d.png)
- View all activity in your region --> View all activity in your area

![Screen Shot 2020-08-25 at 5 13 37 PM](https://user-images.githubusercontent.com/22801026/91240261-59228800-e6f6-11ea-9f0a-fb5e87120c68.png)
- ~~Region --> Chapter?~~

## QA STEPS

* Basically just make sure all of the text changes listed above are correct 😄

**Comment by shaunxp20** [comment]
@rsgonzal 

Just need confirmation on which one to use here: `Region --> Chapter?` and then I think this one will be good to go.

Also, thanks for the screenshots, before/after text, made it easypeasy! 😃

**Comment by rsgonzal** [comment]
hey @shaunxp20 ! Team wants to leave it as region, so no change necessary :)

**Comment by shaunxp20** [comment]
> hey @shaunxp20 ! Team wants to leave it as region, so no change necessary :)

👍 Sounds good, thank you! 😄

**Comment by shaunxp20** [comment]
This one is on the QA site, and some basic QA steps have been added as well. 😄 👍

**Comment by ophelieh** [comment]
Logged in as a Chapter Ambassador and it looks great. I also checked how it looked from the mentor side - looked fine. 

I just have one request -  can we add some help text under "Introduce yourself to your area" to say "This is what participants in your area will see"? 
<img width="1440" alt="Screen Shot 2020-09-10 at 8 28 19 PM" src="https://user-images.githubusercontent.com/32206421/92838420-5e452f80-f3a4-11ea-89fd-726a6a98d6ee.png">

**Comment by ksrebecca** [comment]
Everything worked, I like the text change that Ophelie proposed too.

And I logged in as a student in the same area and looked good from that perspective too.
<img width="699" alt="Screen Shot 2020-09-11 at 7 50 47 AM" src="https://user-images.githubusercontent.com/43972336/92933582-a3e51500-f403-11ea-9921-b1a5f66bd02f.png">

**Comment by shaunxp20** [comment]
Thanks for QAing this, @ophelieh and @ksrebecca! 👍 🙌 🌞 😄 

> I just have one request - can we add some help text under "Introduce yourself to your area" to say "This is what participants in your area will see"?

Great suggestion! I added ticket #2653 for this for the dev team to review. 😄

**Comment by ophelieh** [comment]
Thank you Shaun! :)
